client/src/components/layout/RedBackground.tsx

```diff
@@ -1,13 +1,26 @@
-import React from 'react';
+import React, {useRef, useEffect} from 'react';
 
 
 const RedBackground = () => {
     const leftColor = 'ff594c';
     const rightColor = 'eb144c';
 
+    // Parallax
+    const svg = useRef<SVGSVGElement>(null);
+
+    useEffect(() => {
+        if (!svg.current) return;
+        const handleScroll = () => {
+            svg.current!.style.top = '-' + window.scrollY / 5 + 'px';
+        }
+        document.addEventListener('wheel', handleScroll);
+        return function cleanup() {
+            document.removeEventListener('wheel', handleScroll);
+        }
```

Comment:
Use the `scroll` event since that's the actual event that is dispatched when the scroll position changes. The `wheel` event is dispatched when the scroll wheel moves, but that is not a perfect correlation with scrolling; the scroll wheel is also used for zooming (zooming by pinching with the trackpad dispatches `wheel` events with `ctrlKey` set to true), and the page can be scrolled without using the scroll wheel (using a finger, arrow keys, tab key, URL fragments, and Windows smooth scrolling)